### PR TITLE
switch to new Alinode

### PR DESCRIPTION
**Pull request by JacksonTian**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: config/config.default.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ module.exports = appInfo => {
 
   exports.alinode = {
     enable: true,
-    // default is alinode.aliyun.com
-    server: 'agentserver.alinode.aliyun.com:8080',
+    // default is wss://agentserver.node.aliyun.com:8080
+    server: 'wss://agentserver.node.aliyun.com:8080',
```
Comment (fengmk2) [inline config/config.default.js:20]: 嗯，还是属于 breaking change，我发 major 版本。

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ exports.alinode = {
 };
 ```
 
-Add `appid` and `secret` from http://alinode.aliyun.com/
+Add `appid` and `secret` from https://node.console.aliyun.com/
```
Comment (popomore) [inline README.md:43]: 是不要钱了吗

Comment (fengmk2) [reply README.md:43]: 不要了。

**Comment by fengmk2** [comment]
2.0.0

**Pull request merged**